### ISSUE TITLE
Magnific popup ignores Title and / or Alt tags

### ISSUE THREAD

**Issue by anketwokings** [bolt/bolt#5742]
Hi, 
I used the magnific popup:

```
{{record.image|popup(500, 500, 'r') }}
```

This outputs:

```
<a class="magnific" title="Image: 2016-09/abstract.jpg" href="/thumbs/1000x750r/2016-09/abstract.jpg">
            <img width="500" height="500" alt="Image: 2016-09/abstract.jpg" src="/thumbs/500x500r/2016-09/abstract.jpg">
        </a>
```

Note the **title attribute**. This is the filename. 

But in my contenttypes I have an alt tag for this image:
```
image:
            type: image
            group: media
            label: 'Main image'
            attrib: alt 
```
I would expect the pop-up tag to either:
- be able to specify a field for the alt
- to automatically scan for an alt or a title tag, where title is preferable over alt.

TL;DR:
I want the title under the image in the magnific popup to show the title or alt, and not the filename. :)

Details
-------

 - Relevant Bolt Version: 3.0.12

**Comment by nbehier** [comment]
I didn't test recently, but you could force the title with a fourth arguments : `{{record.image|popup(500, 500, 'r', 'My title') }}` or `{{record.image|popup(500, 500, 'r', record.values.image.title) }}`

**Comment by anketwokings** [comment]
You, dear sir, are absolutely right.

`{{record.image|popup(500, 500, 'r', record.values.image.title) }}` inserts the preferred alt tag where I want it.  

I'll say this is more a documentation issue, and edit the docs for this.

Thank you!

**Comment by anketwokings** [comment]
Closing this because it was possible, all along. :)
